sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -480,6 +525,28 @@ export interface CryptographyOptions extends coreHttp.OperationOptions {}
  */
 export interface GetRandomBytesOptions extends coreHttp.OperationOptions {}
 
+/**
+ * Options for {@link KeyClient.releaseKey}
+ */
+export interface ReleaseKeyOptions extends coreHttp.OperationOptions {
+  /** A client provided nonce for freshness. */
+  nonce?: string;
+
+  /** The {@link KeyExportEncryptionAlgorithm} to for protecting the exported key material. */
+  algorithm?: KeyExportEncryptionAlgorithm;
```

Comment:
I like the name, but should we have the service team change the x-ms-client-name? https://github.com/Azure/azure-rest-api-specs/blob/e7682aa897902920f3a95b2f358b6f7729d18666/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json#L1856

I think what is in swagger now - which changing won't change the wire format - is confusing with our existing `EncryptionAlgorithm` we have in cryptography.

/cc @Lusitanian

Reply:
/cc @herveyw-msft